Utils/github_workflow_scripts/utils.py

```diff
@@ -248,3 +250,24 @@ def get_doc_reviewer(content_roles: dict[str, Any]) -> str:
     if not (reviewer := content_roles.get(DOC_REVIEWER_KEY)):
         raise ValueError("Cannot get doc reviewer")
     return reviewer
+
+
+def get_metadata(pack_dirs: set[str]) -> dict:
+    """
+    Get the pack metadata.
+
+    Args:
+        pack_dirs (set): paths to the packs that were changed
+
+    Return:
+        - pack metadata dictionary
+    """
+    pack_metadata = {}
+
+    for pack_dir in pack_dirs:
+        if pack_metadata := get_pack_metadata(pack_dir):
+            print(f"pack metadata was retrieved for pack {pack_dir}")  # noqa: T201
+        else:
+            print(f'Could not find pack support level for pack {pack_dir}')  # noqa: T201
+
+    return pack_metadata
```

Comment:
This should not return one pack dicy but all the pack dicts that were changed in this PR, maybe one has the TIM tag where the other does not.

Reply:
agreed, added support form multiple packs